src/package-denier/rules.ts

```diff
@@ -0,0 +1,13 @@
+// package-denier/rules.ts
+
+export type PackageName = string;
+export type Comparator = '>' | '>=' | '=' | '<' | '<=';
+export type PackageVersion = string;
+export type Rule = [PackageName, Comparator, PackageVersion];
+
+/*
+  This is the list of packages that are not allowed to be imported.
+ */
+const rules: Rule[] = [['@aws-sdk/client-*', '>', '3.1.0']];
```

Comment:
Should be `3.193.0` I believe thats the last version we've had live without issues inside aws-nock. 
Also should add:
* `got > 11.8.5` 
* `get-port > 5.1.1`
* `antlr4 > 4.9.3`
* `@aws-sdk/credential-provider-ini > 3.193.0`
* `@aws-sdk/credential-provider-node > 3.193.0`
* `@aws-sdk/smithy-client > 3.193.0`
* `@aws-sdk/types >  3.193.0`